### PR TITLE
Install material UI

### PR DESCRIPTION
**Pull request by katerinamartysko**

### CODE REVIEW

**Comment by katerinamartysko** [comment]
[Install materialUI](https://trello.com/c/esN880lB/5-install-materialui)

File: package.json [hunk 1]
```diff
@@ -45,7 +49,7 @@
   "devDependencies": {
     "@typescript-eslint/eslint-plugin": "^5.43.0",
     "@typescript-eslint/parser": "^5.43.0",
-    "eslint": "^8.27.0",
+    "eslint": "8.22.0",
```
Comment (LexaLukovka) [inline package.json]: Please update `eslint` to the latest version

Comment (katerinamartysko) [reply package.json]: Done

Comment (katerinamartysko) [reply package.json]: Done

**Review by LexaLukovka — APPROVED** [review]

**Pull request merged**